package.json

```diff
@@ -31,12 +31,14 @@
     "dependencies": {
         "@docusaurus/core": "^2.4.0",
         "@docusaurus/preset-classic": "^2.4.0",
+        "@docusaurus/theme-mermaid": "^2.4.1",
         "@easyops-cn/docusaurus-search-local": "^0.35.0",
         "@mdx-js/react": "^1.6.22",
         "@node-rs/jieba": "^1.7.0",
         "classnames": "^2.3.2",
         "clsx": "^1.2.1",
         "dotenv": "^16.0.3",
+        "mermaid": "^10.2.3",
```

Comment:
We dont need this dependency as it is handled by `"@docusaurus/theme-mermaid": "^2.4.1",`